### PR TITLE
Upgrade Stack

### LINKED ISSUE #2774: AppVeyor deployment script fails
Currently seems to be logging this:
```
if defined APPVEYOR_REPO_TAG_NAME (set stack_extra_flags="--flag purescript:RELEASE")
echo "stack_extra_flags=%stack_extra_flags%"
"stack_extra_flags="--flag purescript:RELEASE""
stack --jobs=1 --no-terminal test --pedantic %stack_extra_flags%
Invalid option `--flag purescript:RELEASE'
Usage: stack test [TARGET] [--dry-run] [--pedantic] [--fast]
                  [--ghc-options OPTION] [--flag PACKAGE:[-]FLAG]
                  ([--dependencies-only] | [--only-snapshot] |
                  [--only-dependencies]) ([--file-watch] | [--file-watch-poll])
                  [--exec CMD [ARGS]] [--only-configure] [--trace] [--profile]
                  [--[no-]library-profiling] [--[no-]executable-profiling]
                  [--[no-]haddock] [--haddock-arguments HADDOCK_ARGS]
                  [--[no-]open] [--[no-]haddock-deps] [--[no-]copy-bins]
                  [--[no-]prefetch] [--[no-]keep-going] [--[no-]force-dirty]
                  [--[no-]test] [--[no-]rerun-tests]
                  [--test-arguments TEST_ARGS] [--coverage] [--no-run-tests]
                  [--[no-]bench] [--benchmark-arguments BENCH_ARGS]
                  [--no-run-benchmarks] [--[no-]reconfigure]
                  [--[no-]cabal-verbose] [--[no-]split-objs] [--help]
  Shortcut for 'build --test'
Command exited with code 1
```
I'll build it locally & upload for the release for now.

### PR DESCRIPTION
**Pull request by paf31**
Trying to fix #2774

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: '{build}'
 environment:
   # Keep the path as short as possible, just in case.
   STACK_ROOT: c:\s
-  STACK_VER: 1.2.0
+  STACK_VER: 1.3.2
```
Comment (kRITZCREEK) [inline appveyor.yml]: 1.4.0 is the newest version, unless appveyor doesn't have that yet.

Comment (paf31) [inline appveyor.yml]: Ah ok, I just went with what was on my machine 😄

**Comment by paf31** [comment]
This seems to fix the Appveyor build, so I'll merge. As @kRITZCREEK points out, we can update again to 1.4.0, but I don't want to wait for the build again ...

**Comment by garyb** [comment]
Looks like this didn't fix the deployment problem unfortunately. I'm building 0.11.1 manually now.

**Pull request merged**